packages/frontend/amp/components/AdUserSync.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const preBidSrc =
+    'https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/load-cookie.html';
+
+const preBidImg =
+    'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==';
+
+export const AdUserSync: React.FC<{}> = () => {
+    return (
+        <>
+            <amp-iframe
+                data-block-on-consent=""
+                width={1}
```

Comment:
Are interpolations needed for litterals? Could it just be sandbox="allow-scripts" , width=1 (or width="1") ?

Reply:
Will update. Thanks.